metagraph/src/cli/config/config.hpp

```diff
@@ -142,6 +142,8 @@ class Config {
     std::string fasta_anno_comment_delim = UNINITIALIZED_STR;
     std::string header = "";
     std::string host_address;
+    std::string taxonomic_tree;
+    std::string lookup_table;
```

Comment:
find a more descriptive name instead of `lookup_table`

Reply:
replaced with "--lookup-label-taxid"

Reply:
what about `--label-taxid-map`?